Automattic/wp-calypso
# Plans 2023: Implement responsive UI on the plan comparison grid

#### Proposed Changes

This adds the styling needed to make the Plan comparison grid responsive.

Figma design: 2ee4e-pb

#### Testing Instructions



* Go to /start/onboarding-2023-pricing-grid/plans
* Cycle through different window sizes (ex. 375px, 880px, 1500px)
* Plans comparison grid should match the designs in Figma

NOTE: The Feature group expand/collapse feature on mobile hasn't been implemented yet.

<img width="480" alt="Screenshot 2023-01-26 at 15 07 54" src="https://user-images.githubusercontent.com/2749938/214843348-061bd588-5fe1-4920-9dcb-f1f69fc22406.png">
<img width="320" alt="Screenshot 2023-01-26 at 15 07 28" src="https://user-images.githubusercontent.com/2749938/214843355-df749987-03f0-403a-b7ca-c41a4c6d0974.png">
<img width="200" alt="Screenshot 2023-01-26 at 15 07 15" src="https://user-images.githubusercontent.com/2749938/214843358-a47c8e60-f879-4025-900c-b94efda7f521.png">


#### Pre-merge Checklist





Related to #
